source/libsodium/package.d

```diff
@@ -68,62 +68,3 @@ public import libsodium.randombytes_salsa20_random;
 public import libsodium.randombytes_sysrandom;
 public import libsodium.runtime;
 public import libsodium.utils;
-
-version (unittest)
-shared static this ()
-{
-    auto res = sodium_init();
-    assert(res >= 0);
-}
-
-/// https://libsodium.gitbook.io/doc/secret-key_cryptography/authenticated_encryption
-unittest
```

Comment:
Having this in a single module generate better documentation though

Reply:
I just planning also add idiomatic bindings calls, and then rewrite tests to use them

Reply:
Those are idiomatic binding calls. They are taken straight from the documentation.

Reply:
> Those are idiomatic binding calls

I mean idiomatic wrappers as in the D language: without pointers, with types instead of byte arrays etc. It have ~zero cost, but makes usage more convient and safer.

Its usage will not be forced, but it is suitable for unittesting.